client/hardening.ts

```diff
@@ -33,16 +33,32 @@ if (!CSRFDone) {
 
 // Custom fetch request that injects the CSRF token into the body
 export async function fetchWithCSRF (url: string, requestInit: RequestInit): Promise<Response> {
-	const body = requestInit.body ?? {};
+	const body = JSON.parse(requestInit.body as string) ?? {};
```

Comment:
didn't this change break it?

Reply:
This change makes update work but breaks signout without a body part in the request

Reply:
I think since #25 is being merged soon, this change is redundant and instead the version from #25 could be used here.